default.nix

```diff
@@ -32,15 +32,39 @@ let
 
     server = hself.callCabal2nix "server" (src + "/server") {};
 
-    jose = hself.callHackage "jose" "0.8.5" {};
+    jose = hself.jose_0_9;
 
     base64-bytestring = hself.base64-bytestring_1_2_1_0;
 
+    aeson = hself.aeson_2_0_3_0;
```

Comment:
As already discussed on calls, we should add another CI target to make sure aeson 1.x works